audiotoolbox.sh

```diff
@@ -37,4 +37,20 @@ audioclip(){
 	rm "temp_$1"
 }
 
-
+flacer(){
+	echo Do you want a funny version? [y/N]
+	read $opt;
+	
+	if [ "$opt" == "N" ]; then
+	ffmpeg -hide_banner -i "$1" -c:a flac -af aformat=s16:44100 "${1%.*}.flac"
+	echo -e "\e[1;33mConversion complete: "${1%.*}.flac" in $PWD\e[0m"
```

Comment:
Repeated message that is displayed regardless of process exit code, recommended to move it outside the conditional statement blocks, near the function closing bracket.

Reply:
This problem will be resolved jointly with the removal of the parts that require lolcat.